motile/utils.py

```diff
@@ -42,6 +42,6 @@ def get_tracks(graph, require_selected=False, selected_attribute='selected'):
         TrackGraph(
             graph_data=graph.subgraph(g).copy(),
             frame_attribute=graph.frame_attribute,
-            roi=graph.roi)
+        )
```

Comment:
👀 @funkey  ... mypy caught this as an invalid keyword argument to `TrackGraph`, and it appears correct... but please confirm

Reply:
Ah, yes, thanks for spotting that. `utils.py` needs some love.